Mathlib/SetTheory/Cardinal/Basic.lean

```diff
@@ -1029,6 +1030,15 @@ theorem bddAbove_range_comp {ι : Type u} {f : ι → Cardinal.{v}} (hf : BddAbo
   rw [range_comp]
   exact bddAbove_image g hf
 
+/-- The type of cardinals in universe `u` is not `Small.{u}`. This is a version of the Burali-Forti
+paradox. -/
+theorem not_small_cardinal : ¬ Small.{u} Cardinal.{u} := by
```

Comment:
Would it be hard to change this to `¬Small.{u} Cardinal.{max u v}`, like `not_small_ordinal`?

Reply:
Not at all, I've performed the generalization.